draft.xml

```diff
@@ -101,500 +88,440 @@
-<eref target="http://tools.ietf.org/html/rfc4291#section-2.3">section 2.3</eref>
-of <xref target="RFC4291"/> for IPv6.
-            </t>
-            <t>
-Examples include "192.0.2.1" and "192.0.2.0/24" for IPv4 and "2001:db8::1"
-and "2001:db8::/32" for IPv6.
-            </t>
+            <t>REQUIRED. Each IP prefix field MUST be either a single IP
+            address or an IP prefix in CIDR notation in conformance with <eref
+            target="http://tools.ietf.org/html/rfc4632#section-3.1">section
+            3.1</eref> of <xref target="RFC4632"/> for IPv4 or <eref
+            target="http://tools.ietf.org/html/rfc4291#section-2.3">section
+            2.3</eref> of <xref target="RFC4291"/> for IPv6.</t>
+
+            <t>Examples include "192.0.2.1" and "192.0.2.0/24" for IPv4 and
+            "2001:db8::1" and "2001:db8::/32" for IPv6.</t>
           </section>
 
           <section title="Country">
-            <t>
-OPTIONAL.  The country field, if non-empty, MUST be a 2 letter ISO country
-code conforming to ISO 3166-1 alpha 2 <xref target="ISO.3166.1alpha2"/>.
-Parsers SHOULD treat this field case-insensitively.
-            </t>
-            <t>
-Examples include "US" for the United States, "JP" for Japan, and "PL" for
-Poland.
-            </t>
+            <t>OPTIONAL. The country field, if non-empty, MUST be a 2 letter
+            ISO country code conforming to ISO 3166-1 alpha 2 <xref
+            target="ISO.3166.1alpha2"/>. Parsers SHOULD treat this field
+            case-insensitively.</t>
+
+            <t>Examples include "US" for the United States, "JP" for Japan,
+            and "PL" for Poland.</t>
           </section>
 
           <section title="Region">
-            <t>
-OPTIONAL.  The region field, if non-empty, MUST be a ISO region code
-conforming to ISO 3166-2 <xref target="ISO.3166.2"/>.  Parsers SHOULD
-treat this field case-insensitively.
-            </t>
-            <t>
-Examples include "ID-RI" for the Riau province of Indonesia and "NG-RI" for
-the Rivers province in Nigeria.
-            </t>
+            <t>OPTIONAL. The region field, if non-empty, MUST be a ISO region
+            code conforming to ISO 3166-2 <xref target="ISO.3166.2"/>. Parsers
+            SHOULD treat this field case-insensitively.</t>
+
+            <t>Examples include "ID-RI" for the Riau province of Indonesia and
+            "NG-RI" for the Rivers province in Nigeria.</t>
           </section>
 
           <section title="City">
-            <t>
-OPTIONAL.  The city field, if non-empty, SHOULD be free UTF-8 text,
-excluding the comma (',') character.
-            </t>
-            <t>
-Examples include "Dublin", "New York", and "São Paulo" (specifically
-"S" followed by 0xc3, 0xa3, and "o Paulo").
-            </t>
+            <t>OPTIONAL. The city field, if non-empty, SHOULD be free UTF-8
+            text, excluding the comma (',') character.</t>
+
+            <t>Examples include "Dublin", "New York", and "S?o Paulo"
```

Comment:
São Paulo

Reply:
As above -- see the rendered (current draft) at https://tools.ietf.org/html/draft-google-self-published-geofeeds-02#section-2.1.1.4

It renders as Sao Paulo -- this is obviously sub-optimal (and, yes, my S?o Paulo is obviously even more broken!), but we cannot get São Paulo into an RFC easily at the moment... 
I can try craft some text along the lines of "Examples include Sao Paulo (with the correct dicritic) would be "S" followed by 0xc3, 0xa3, and "o Paulo")" or something.

Reply:
As above -- by the time this is published, this should be addressable. 
a5ec5efadd76c31a5a6b5e603b0f28289a5d0830